(DOCSP-22032): Add details for Swift SDK offline login (#1846)
mongodb/docs-realm: source/sdk/swift/examples/configure-and-open-a-realm.txt

.. include:: /examples/generated/code/start/Sync.codeblock.specify-download-behavior.swift.code-block.rst

.. _ios-open-a-synced-realm-offline:

Open a Synced Realm Offline
~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. include:: /includes/offline-login.rst

> This only works if you do not require your Realm app to ``always`` :ref:`download

Reviewer: ```suggestion
You can only open a synced realm offline if you do not require your realm to ``always`` :ref:`download
```

Reviewer: first change is just a copy edit. second, changing "Realm app" -> "realm" is b/c i think we're talking abt an individual realm database file here, not the whole Realm app.

Author: Fair point. I meant "client app" here, but with our naming, it's really confusing to call a client app a "Realm app" because there's also a "Realm app" backend. Fixing.
